spec/api/commentable_spec.rb

```diff
@@ -32,6 +32,11 @@ def thread_result(commentable_id, params={})
         threads.index{|c| c["body"] == "can anyone help me?"}.should_not be_nil
         threads.index{|c| c["body"] == "it is unsolvable"}.should_not be_nil
       end
+      it "returns standalone threads if explicitly requested" do
+        threads = thread_result "question_1", context: "standalone"
```

Comment:
Sorry, looks like I missed the later comments. This test isn't fetching a single thread, but a group of threads associated with a commentable ("question_1"). These requests need to have context specified if it is something other than "course". Jim is correct that the ability to fetch a single thread by id has not changed.